lib/entities/provider_types.dart

```diff
@@ -80,7 +80,7 @@ class ProvidersHelper {
       case WalletType.none:
       case WalletType.haven:
       case WalletType.decred:
-        return [];
+        return [ProviderType.askEachTime, ProviderType.onramper];
```

Comment:
Does this also change the behavior for `WalletType.none` and `WalletType.haven`?

Reply:
Updated.